plop-templates/component.tsx.hbs

```diff
@@ -2,9 +2,19 @@ import React from "react";
 import Styled{{name}} from "./{{name}}.styles";
 
 export interface {{name}}Props {
+  /**
+  * `children` can be any `ReactNode`
+  */
   children?: React.ReactNode;
 };
 
+/**
+ * 
+ * `{{name}}` describe what I do 
```

Comment:
Suggestion; nudge users of the template to provide extra context not available from reading the code.

For example; in addition to prompting to describe what the component does, prompt to describe why it exists, how it's used etc.

Reply:
Yep - I think we can make that better 

>Describe what it does: ✅


>Describe why it exists: 

Question; is this overkill for UI components?


>Describe how its used: 

Question; Do you think a link to the storybook is enough here?
`* Component Demo: [{{name}}](https://ui.govconnex...)`

Reply:
Yeah I guess storybook itself demonstrates the usage of the component, so a link is convenient.

Reply:
I think where I was going with "How it's used", is more about summarising common use cases, rather than the technical usage.

Reply:
You're right, it's probably overkill though, we want the key information that's easy to add rather than a small story for each component. 😁